tests/js/spec/views/alerts/create.spec.jsx

```diff
@@ -338,16 +253,14 @@ describe('ProjectAlertsCreate', function () {
         body: TestStubs.ProjectAlertRule(),
       });
 
-      await waitFor(() => {
-        expect(memberActionCreators.fetchOrgMembers).toHaveBeenCalled();
-      });
-
       // Change target environment
       await selectEvent.select(screen.getByText('All Environments'), ['production']);
 
       // Change actionMatch and filterMatch dropdown
-      await selectEvent.select(screen.getAllByText('all')[0], ['any']);
-      await selectEvent.select(screen.getAllByText('all')[0], ['any']);
+      const allDropdowns = screen.getAllByText('all');
+      expect(allDropdowns).toHaveLength(2);
+      await selectEvent.select(allDropdowns[0], ['any']);
+      await selectEvent.select(allDropdowns[1], ['any']);
```

Comment:
do we need multiple await here?

Reply:
we could promise.all them 🤷 but yes its two separate dropdown actions